portability-core/src/main/java/org/dataportabilityproject/cloud/google/GoogleCloudModule.java

```diff
@@ -59,4 +60,23 @@ GoogleCredentials getCredentials(CommonSettings commonSettings) throws GoogleCre
     }
   }
 
+  /**
+   * Get project ID from environment variable and validate it is set.
+   *
+   * @throws IllegalArgumentException if project ID is unset
+   */
+  @Provides ProjectId getProjectId() {
```

Comment:
Is this only used for local talking to cloud or also when deployed on the docker image. If the former, we might want to created a separate module and annotation for local only.

Reply:
It's used for the Docker image too. Not a local-only thing